src/App.tsx

```diff
@@ -20,12 +29,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => {
+                  setSearch(event.target.value.trim());
```

Comment:
Better to create a function handler for it

Reply:
agree

Reply:
@LesyaSvk don't be afraid to reject when there are meaningful issues